ksigner/src/jvmTest/kotlin/com/novoda/ksigner/hash/DecodeRequestTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.novoda.ksigner.hash
+
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class DecodeRequestTest {
```

Comment:
This is the test has been ported from the [aws-v4-signer java lib](https://github.com/lucasweb78/aws-v4-signer-java/blob/master/src/test/java/uk/co/lucasweb/aws/v4/signer/Sha256Test.java) to verify both, the SHA-256 implementation and the hex function.